app/controllers/rubcop_test_controller.rb

```diff
@@ -1,17 +1,18 @@
-class RubocopController  < ApplicationController
-	def initialize
-		print "this is so bad patern"
-	end
+class RubocopController < ApplicationController
+  def initialize
+    print 'this is so bad patern'
+  end
 
-	def print_fib(range)
-		range.each do |n|
-			p fib(n)
-		end
-	end
+  def print_fib(range)
+    range.each do |n|
+      p fib(n)
+    end
+  end
 
-	private
-		def fib(n)
-			return 1 if n <= 1
-			fib(n-2) + fib(n-1)
-		end
-end
\ No newline at end of file
+  private
+
+  def fib(n)
```

Comment:
__[reek]__
* [FeatureEnvy] refers to n more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)